index.html

```diff
@@ -48,22 +47,192 @@ <h2 style="font-size: clamp(30px, 4vw, 45px); font-weight: 700;">
+        </p>
+      </div>
+      <div class="socials">
+        <h4 class="letsconnect">LET'S CONNECT</h4>
+        <ul class="socials-mainlist">
+          <ol class="socials-list">
             <a href="#"><img src="./icons/github.png" alt="github icon" /></a>
           </ol>
-          <ol>
+          <ol class="socials-list">
             <a href="#"><img src="./icons/linkedin.png" alt="linkedin icon" /></a>
           </ol>
-          <ol>
+          <ol class="socials-list">
             <a href="#"><img src="./icons/twitter.png" alt="twitter icon" /></a>
           </ol>
-          <ol>
+          <ol class="socials-list">
             <a href="#"><img src="./icons/medium.png" alt="medium icon" /></a>
           </ol>
         </ul>
       </div>
+      <button class="resume">Get my resume</button>
+      <div>
+        <h3>Languages</h3>
+        <div class="lang">
+          <div class="java">
+            <img class="lang-img" src="./icons/javascript.png" alt="" />
+            Javascript
+            <!-- <p class="lang-text">Javascript</p> -->
+          </div>
+          <div class="html">
+            <img class="lang-img" src="./icons/html.png" alt="" />
+            HTML
+            <!-- <p class="lang-text">HTML</p> -->
+          </div>
+          <div class="css">
+            <img class="lang-img" src="./icons/css.png" alt="" />
+            CSS
+            <!-- <p class="lang-text">CSS</p> -->
+          </div>
+        </div>
+        <!-- Large button groups (default and split) -->
+        <div class="framework">
+          <div class="btn-group">
+            <button class="btn btn-secondary btn-lg" type="button" style="background-color: #e5e5e5; border: 1px; color: #000;">Frameworks</button>
+            <button type="button" class="btn btn-lg btn-secondary dropdown-toggle dropdown-toggle-split" data-bs-toggle="dropdown" aria-expanded="false" style="background-color: #e5e5e5; border: 1px; color: #000; margin-left: 40px;">
+              <span class="visually-hidden">Toggle Dropdown</span>
+            </button>
+            <ul class="dropdown-menu">
+              ...
+            </ul>
+          </div>
+        </div>
+        <div class="framework">
+          <div class="btn-group">
+            <button class="btn btn-secondary btn-lg" type="button" style="background-color: #e5e5e5; border: 1px; color: #000;">Skills</button>
+            <button type="button" class="btn btn-lg btn-secondary dropdown-toggle dropdown-toggle-split" data-bs-toggle="dropdown" aria-expanded="false" style="background-color: #e5e5e5; border: 1px; color: #000; margin-left: 110px;">
+              <span class="visually-hidden">Toggle Dropdown</span>
+            </button>
+            <ul class="dropdown-menu">
+              ...
+            </ul>
+          </div>
+        </div>
+      </div>
```

Comment:
This section needs some more work to match the design. You can do it 💯 See below for the differences 👇🏽 

## Figma
![Screenshot 2023-02-17 at 21 03 36](https://user-images.githubusercontent.com/45739963/219825977-5415def8-6d0a-4687-b367-ffe47707d554.png)

## Your design
![Screenshot 2023-02-17 at 21 03 56](https://user-images.githubusercontent.com/45739963/219825994-299a689b-e385-4bc9-8c83-a5c4b327929e.png)